new/LIT6469HomilyKahenat.xml

```diff
@@ -0,0 +1,62 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6469HomilyKahenat" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">መጽሐፈ፡ ድርሳን፡ ዘ፳ወ፬ካህናተ፡ ሰማይ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Maṣḥafa dǝrsān za-24kāhǝnāta samāy</title>
```

Comment:
I would probably add a space between the numerals and the following word: ዘ፳ወ፬ ካህናተ፡ and za-24 kāhǝnāta

Reply:
Definitely looks better. I reproduced the original ortography.